### ISSUE TITLE
Invisibility of social media links

### ISSUE THREAD

**Issue by Kajalmehta29** [dohinaf/basic-icecream-website#360]
The social media links at the end are not visible properly due to less color contrast . I want to work on this issue.
Please assign this issue to me under gssoc-extd and hacktoberfest.
